.github/workflows/test.yml

```diff
@@ -0,0 +1,94 @@
+on: [push]
+name: tests
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    services:
+      # Label used to access the service container
+      postgres:
+        # Docker Hub image
+        image: postgres
+        # Provide the password for postgres
+        env:
+          POSTGRES_PASSWORD: postgres
+        # Set health checks to wait until postgres has started
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.6]
+        tox_env: [py36]
+        galaxy_version: [dev,
+                         release_20.09,
+                         release_20.05,
+                         release_20.01,
+                         release_19.09,
+                         release_19.05,
+                         release_19.01,
+                         release_18.09,
+                         release_18.05,
+                         release_18.01,
+                         release_17.09]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+          galaxy-version: ${{ matrix.galaxy_version }}
```

Comment:
```suggestion
          python-version: ${{ steps.get-python-version.outputs.python_version }}
```

Reply:
reverted this one, because it doesn't seem to work

Reply:
Mmmh, this was connected to another suggestion that apparently got lost, I'll re-add it later.